spec/features/jobs/user_edits_existing_job_spec.rb

```diff
@@ -6,12 +6,12 @@
     category = Category.create!(title: "Category")
     job = company.jobs.create!(title: "Developer", level_of_interest: 90, city: "Denver", category_id: category.id)
 
-    visit edit_company_job_path(company, job)
+    visit edit_job_path(job)
 
     fill_in "job[title]", with: "BBaller"
     click_button "Update"
 
```

Comment:
This still has a problem here with automatically trying company_job_path from the job _form.html.erb but the fix of removing @company causes new path to fail